modules/architecture-machine-roles.adoc

```diff
@@ -26,7 +26,7 @@ Machine sets are groupings of machine resources under the `machine-api` namespac
 [id="defining-masters_{context}"]
 == Cluster masters
 
-In a Kubernetes cluster, the control plane nodes (also known as the master nodes) run services that are required to control the Kubernetes cluster. In {product-title}, the control plane machines are the control plane. They contain more than just the Kubernetes services for managing the {product-title} cluster. Because all of the machines with the control plane role are control plane machines, the terms _master_ and _control plane_ are used interchangeably to describe them. Instead of being grouped into a machine set, control plane machines are defined by a series of standalone machine API resources. Extra controls apply to control plane machines to prevent you from deleting all control plane machines and breaking your cluster.
+In a Kubernetes cluster, the control plane nodes run services that are required to control the Kubernetes cluster. In {product-title}, the control plane machines are the control plane. They contain more than just the Kubernetes services for managing the {product-title} cluster. Because all of the machines with the control plane role are control plane machines, the terms _master_ and _control plane_ are used interchangeably to describe them. Instead of being grouped into a machine set, control plane machines are defined by a series of standalone machine API resources. Extra controls apply to control plane machines to prevent you from deleting all control plane machines and breaking your cluster.
```

Comment:
+1 to removing that sentence

Reply:
I feel like the sentence is still relevant if the role is defined by its existing name, `master`:

```Because all of the machines with the master role are control plane machines, the terms _master_ and _control plane_ are used interchangeably to describe them.```

This comment is pending whether we can confirm what roles are available :slightly_smiling_face: